bin/yarn

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
We can delete this file as well since we don't we don't use yarn (I think? Honestly I don't know but I think yarn is brand new in Rails 5.1 so presumably we're not using it.)